City-of-Helsinki/drupal-helfi-rekry
# UHF-5287 Change organization and organization name fields other way around

# UHF-5287 Change organization and organization name fields other way around [UHF-5287](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5287)


Change organization and organization name fields priting so that organization field is where organization name is and the other way around.

## What was done


* Changed the template, styles and configuration so that the fields change places neatly.

## How to install

* Checkout this branch on Rekry instance
  * `git pull origin UHF-5287_change_organization_and_organization_name_fields`
  * `make drush-cr && make drush-cim && make drush-cr`

## How to test


* Make sure that the fields are now different way around than in here: https://nginx-rekry-test.agw.arodevtest.hel.fi/fi/test-avoimet-tyopaikat/avoimet-tyopaikat/sote-03-988-22
* See the screenshot.

## Designers review


* [ ] This PR has been visually reviewed by a designer (Sami)

<img width="1387" alt="image (2)" src="https://user-images.githubusercontent.com/2276077/171848349-848029b9-fc64-4267-b498-73df1a8a38d0.png">
